src/transformers/Streamable.js

```diff
@@ -0,0 +1,25 @@
+import { URL } from 'url';
+import fetch from 'node-fetch';
+
+const getTrimmedPathName = pathname => pathname.replace(/^\/|\/+$/g, '');
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+  const trimmedPathName = getTrimmedPathName(pathname).split('/');
+
+  return (
```

Comment:
I think you should return the following to express more the meaning of what we're checking:

```js
  return (
    ['streamable.com', 'www.streamable.com'].includes(host) &&
    trimmedPathName.length > 0 &&
    trimmedPathName.length <= 3 &&
    (
      (
        trimmedPathName.length === 1 &&
        !['/documentation', '/login', '/recover', '/settings', '/signup'].includes(pathname)
      ) ||
      ['e', 'g', 'o', 's', 't'].includes(trimmedPathName[0])
    )
  );
```